brian2/units/fundamentalunits.py

```diff
@@ -644,20 +642,15 @@ def get_dimensions(obj):
 
     Returns
     -------
-    dim: `Dimension`
+    dim : `Dimension`
         The physical dimensions of the `obj`.
     """
     try:
         return obj.dim
     except AttributeError:
         # The following is not very pretty, but it will avoid the costly
         # isinstance check for the common types
-        # Workaround for Python 3 which no longer has a long type
-        try:
-            long
-        except NameError:
-            long = int
-        if (type(obj) in [int, long, float, np.int32, np.int64,
+        if (type(obj) in [int, float, np.int32, np.int64,
```

Comment:
Hi @mstimberg, are you happy with this?

Reply:
I'll have a go at the msvc thing, I think it is easier to do than to explain :)

I'm happy with the `int`/`long` change in `fundamentalunits.py`, don't see any issue there.